.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-    "editor.insertSpaces": true
-}
\ No newline at end of file
+    "editor.insertSpaces": true,
+    "files.trimTrailingWhitespace": false
```

Comment:
I can remove this. I think I'm using [this extension](https://marketplace.visualstudio.com/items?itemName=samverschueren.final-newline) that adds a new line and tests care about whitespace.